harness/determined/deploy/gcp/cli.py

```diff
@@ -176,6 +228,43 @@ def parse(s: str) -> Tuple[str, str]:
     None,
     "GCP help",
     [
+        Cmd(
+            "list",
+            handle_list,
+            "list gcp cluster",
+            [
+                ArgGroup(
```

Comment:
nit: ArgGroups with a single argument doesn't sound very useful. does it change anything we don't use arg groups?

Reply:
It works without the ArgGroup. fixed it!